ova/generate_ova.sh

```diff
@@ -34,12 +36,11 @@ help () {
 
     echo
     echo "Usage: $0 [OPTIONS]"
-    echo
-    echo "  -b, --build            [Required] Build the OVA and OVF."
     echo "  -v, --version          [Required] Version of wazuh to install on VM."
-    echo "  -e, --elastic-version  [Required] Elastic version to download inside VM."
-    echo "  -r, --repository       [Required] Status of the packages [stable/unstable]"
-    echo "  -d, --directory        [Optional] Where will be installed manager. Default /var/ossec"
+    echo "  -o, --opendistro       [Required] Version of opendistro"
+    echo "  -e, --elasticsearch    [Required] Filebeat version to download inside VM."
+    echo "  -r, --repository       [Required] Status of the packages [prod/dev]"
```

Comment:
```suggestion
    echo "  -r, --repository       [Required] Select the software repository [prod/dev]."
```

Reply:
Done in 528feec208a367ba616fb87ea4237096d0681494